spec/chief/integration_spec.rb

```diff
@@ -25,7 +25,31 @@ def call
     end
   end
 
+  class ExampleWithoutChiefResultCommand < Command
+    def call
+      :some_value
+    end
+  end
+
+  class ExampleWithChiefResultPassThroughCommand < Command
+    def call
+      ExampleFailingCommand.call(:some_value)
+    end
+  end
+
   RSpec.describe Command do
+    describe "Commands that don't return a Chief::Result object" do
+      it 'should raise an error' do
+        expect { ExampleWithoutChiefResultCommand.call }.to raise_error(RuntimeError)
```

Comment:
@smeriwether What do you think about returning a more specific error class here? RuntimeErrors can be misleading when trying to debug.

Reply:
I thought about that. My reasoning was this:

1. The RuntimeError's message is descriptive
2. Nobody should ever rescue from this error

I don't have strong feelings about this if you do I will certainly change it.

Reply:
On second thought, I have the same thoughts as described ^ but I think the error message should be changed. I will think of a better message.

Reply:
The new error message is:
`{class_name} must call success! or fail! and return the result`

which I am happier about.